### DISCUSSION TITLE
Should Service Entity have an unique index for Service Name and Version?

### DISCUSSION THREAD

**Discussion by Sgitario** [halkyonio/primaza-poc#56] [Q&A] [answered]
The `Claim` entity has a field called `serviceRequested`: https://github.com/halkyonio/primaza-poc/blob/863892e0818885b7904dd5081e2629b5ddb7827a/servicebox-app/src/main/java/io/halkyon/model/Claim.java#L28

In the miro board, we can see that this field is mapped to the service name and the service version, so I think we should add a unique index of these two fields within the Service entity. 
Wdyt? @cmoulliard @aureamunoz

**Comment by cmoulliard** [comment, marked as answer]
> add a unique index of these two fields within the Service entity.

+1

**Comment by aureamunoz** [comment]
Oups, I thought having answered to this. I agree also with adding the index :+1:

**Comment by Sgitario** [comment]
Will apply these changes in https://github.com/halkyonio/primaza-poc/pull/77.
